lib/create-entity-finder-api.js

```diff
@@ -28,6 +28,21 @@ function getEntityInDataset(entityByDomain, entityByRootDomain, originOrURL) {
   return undefined
 }
 
+function getProductInDataset(entityByDomain, entityByRootDomain, originOrURL) {
```

Comment:
can you add all this to the typedefs? 
https://github.com/patrickhulce/third-party-web/blob/master/lib/index.d.ts